Geometry/MTDCommonData/src/ETLNumberingScheme.cc

```diff
@@ -0,0 +1,110 @@
+#include "Geometry/MTDCommonData/interface/ETLNumberingScheme.h"
+#include "DataFormats/ForwardDetId/interface/ETLDetId.h"
+
+#include <iostream>
+
+//#define DEB_NUM
+
+ETLNumberingScheme::ETLNumberingScheme() : 
+  MTDNumberingScheme() {
+  edm::LogInfo("MTDGeom") << "Creating ETLNumberingScheme";
+}
+
+ETLNumberingScheme::~ETLNumberingScheme() {
+  edm::LogInfo("MTDGeom") << "Deleting ETLNumberingScheme";
+}
+
+uint32_t ETLNumberingScheme::getUnitID(const MTDBaseNumber& baseNumber) const {
+
+  const uint32_t nLevels ( baseNumber.getLevels() ) ;
+
+#ifdef DEB_NUM
+  edm::LogInfo("MTDGeom") << "ETLNumberingScheme geometry levels = " << nLevels;
+#endif
+  
+
+  if( 11 > nLevels )
+  {   
+     edm::LogWarning("MTDGeom") << "ETLNumberingScheme::getUnitID(): "
```

Comment:
This caused the D19 step1 logfile to explode

Reply:
yes, I see it, this is odd, scenario D19 should not be minimally affected

Reply:
ok, understood, it is coming from reusing the SimG4CMS interface. This scenario has obviously no future, that MTD description will not be used any more. But if we want to maintain it (because it was used for some production) I need to clone the SensitiveDetector interface.